storage/src/main/java/org/apache/uniffle/storage/handler/impl/HadoopShuffleDeleteHandler.java

```diff
@@ -52,7 +56,7 @@ public void delete(String[] storageBasePaths, String appId, String user) {
       while (!isSuccess && times < retryMax) {
         try {
           FileSystem fileSystem = HadoopFilesystemProvider.getFilesystem(user, path, hadoopConf);
-          fileSystem.delete(path, true);
+          delete(fileSystem, path, shuffleServerId);
```

Comment:
Do you having the global var to get the shuffleServerId ?

Reply:
> Do you having the global var to get the shuffleServerId ?

Yes.
https://github.com/apache/incubator-uniffle/blob/30bf8dcc43eb397e737a39b151b739316e84e1d8/server/src/main/java/org/apache/uniffle/server/ShuffleServer.java#L143